src/main/resources/templates/events.html

```diff
@@ -2,11 +2,17 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org"
       xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity3">
 <head>
-    <title>Register New Events</title>
+    <title>Events</title>
 </head>
 <body>
-    <div th:replace="header">...</div>
-    <div class="container ocdu-container"> this is events view </div>
-    <div th:replace="footer :: footer"></div>
+<div th:replace="header">...</div>
+<div id="events-registration-container" class="container ocdu-container">
+    <div id="event-registration-div" class="jumbotron container-fluid">
+    </div>
+</div>
+
+
+<script th:src="@{/events-registration.js}"></script>
+<div th:replace="footer :: footer"></div>
 </body>
-</html>
+</html>
```

Comment:
http://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file